Framework/Kernel/test/FileDescriptorTest.h

```diff
@@ -130,6 +133,11 @@ class FileDescriptorTest : public CxxTest::TestSuite {
     TS_ASSERT_EQUALS(0, stream.tellg());
   }
 
+  void testEmptyFile() {
```

Comment:
Since were not using this for any other test can we move `m_emptyFilePath` into here as a local variable

Additionally, you can use [FileFinder](https://github.com/mantidproject/mantid/blob/3fe5e56b7c926586bcae8141a80a6175ca4c0a6c/Framework/API/test/FileFinderTest.h#L116) rather than building Poco Paths. It's less boilerplate and avoids bugs in test code around handling paths

Reply:
On discussion it was pointed out this is in Kernel so we cannot use FileFinder from API

Reply:
MantidAPI isn't currently available in the KernelTest project and it might be overkill to add it just for this one usage.